src/MediaStreamTrack.ts

```diff
@@ -107,7 +109,11 @@ export default class MediaStreamTrack extends EventTarget<MediaStreamTrackEventM
             throw new Error('Only implemented for video tracks');
         }
 
-        WebRTCModule.mediaStreamTrackSwitchCamera(this.id);
+        const switchImpl = async () => {
```

Comment:
Alright, this is what I meant. Can we simply use the existing settings, flip the facing mode and call applyConstraints with just that constraint?